engine/collection/ingest/config.go

```diff
@@ -27,6 +29,7 @@ func DefaultConfig() Config {
 		MaxGasLimit:           flow.DefaultMaxGasLimit,
 		AllowUnknownReference: false,
 		CheckScriptsParse:     true,
+		MaxAddressIndex:       1_000_000,
```

Comment:
Just to get an idea, do we know what's the current index in Mainnet/Testnet ?

Reply:
Posted in #flow-integration to ask. There should be ~13,000 from CoinList, and I think something like 10-20k-ish from Dapper.

Reply:
Checked and 2 hours ago we had 27272 accounts.